tracpro/contacts/models.py

```diff
@@ -142,7 +138,8 @@ class Meta:
 
     def __init__(self, *args, **kwargs):
         self._data_field_values = kwargs.pop('_data_field_values', None)
-        super(Contact, self).__init__(*args, **kwargs)
+        kwargs.pop('groups', None)
```

Comment:
From contacts/utils.py part of sync_pull_contacts(), starting at line 102: 

`          # create new contact from kwargs
            new_contact = Contact(**kwargs)
            new_contact.save()

            # add all the related groups to
            new_contact.groups = kwargs['groups']`

Here, I've sent the kwargs to create a new contact, but `groups` is not a valid keyword for this __init__ function, so I pop it out of the kwargs.  and then add it into the new_contact.  I didn't really enjoy doing this :), but didn't want to mess this function up if it were being used elsewhere.  I was entertaining adding a `create` class method instead.  Somehow I thought that that would do the job better.

It seems that I maybe should pop the kwargs out on line 101 of contacts/utils.py instead, since I'm really targeting that functionality.